cmd/auth/main.go

```diff
@@ -51,24 +51,28 @@ type config struct {
 	KetoWriteHost string        `env:"MF_KETO_WRITE_REMOTE_HOST"     envDefault:"mainflux-keto"`
 	KetoWritePort string        `env:"MF_KETO_WRITE_REMOTE_PORT"     envDefault:"4467"`
 	LoginDuration time.Duration `env:"MF_AUTH_LOGIN_TOKEN_DURATION"  envDefault:"10h"`
-	JaegerURL     string        `env:"MF_JAEGER_URL"                 envDefault:"localhost:6831"`
 }
 
 func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	g, ctx := errgroup.WithContext(ctx)
 
 	// Create auth service configurations
-	cfg := config{}
-	if err := env.Parse(&cfg); err != nil {
+	c := config{}
+	if err := env.Parse(&c); err != nil {
 		log.Fatalf("failed to load %s configuration : %s", svcName, err)
 	}
 
-	logger, err := mflog.New(os.Stdout, cfg.LogLevel)
+	logger, err := mflog.New(os.Stdout, c.LogLevel)
 	if err != nil {
 		log.Fatalf("failed to init logger: %s", err)
 	}
 
+	cfg := server.Config{}
+	if err := env.Parse(&cfg, env.Options{Prefix: envPrefix}); err != nil {
+		log.Fatalf("failed to load %s server configuration: %s", svcName, err)
+	}
```

Comment:
the other server.config's are for http server and grpc server both of which require a separate port, thus can't use this struct variables.

Reply:
@arvindh123 I would not default to `1`, instance ID should be in the UUID format, because you will not be capable to assure incremental numeration in the distributed system (two instances launched at the same in other parts of the world might get the same ID number - this is exactly what UUID can prevent).